postgres-failover-slots/docker-compose.yaml

```diff
@@ -0,0 +1,109 @@
+services:
+  pgbouncer:
+    image: edoburu/pgbouncer:latest
+    environment:
+      - DB_HOST=postgres_replica
```

Comment:
Should't this point to the `postgres_primary` at the first start?

Reply:
Indeed it should. I had changed it while running through the steps and then committed it. Good spot!